lib/perl/Genome/Model/Tools/EpitopePrediction/NetmhcPipeline.pm

```diff
@@ -0,0 +1,371 @@
+package Genome::Model::Tools::EpitopePrediction::NetmhcPipeline;
+
+use strict;
+use warnings;
+
+use Genome;
+use Workflow::Simple;
+
+class Genome::Model::Tools::EpitopePrediction::NetmhcPipeline {
+    is => 'Command::V2',
+    doc => 'Run the netmhc portion of the epitope binding prediction pipeline',
+    has_input => [
+        output_directory => {
+            is => 'Text',
+            doc => 'the directory where you want results stored',
+        },
+        somatic_variation_build => {
+            is => 'Genome::Model::Build::SomaticVariation',
+            is_optional => 1,
+            doc => 'The somatic variation build to use for analysis',
+        },
+        input_fasta_file => {
+            is => 'Text',
+            doc => 'The fasta file with variant sequences for epitope binding prediction',
+        },
+        alleles => {
+            is => 'Text',
+            doc => 'A list of allele names to be used for epitope prediction with NetMHC',
+            is_many => 1,
+        },
+        epitope_length => {
+            is => 'Text',
+            doc => 'Length of subpeptides to predict with NetMHC',
+        },
+        netmhc_version => {
+            is => 'Text',
+            doc => 'The NetMHC version to use',
+            valid_values => ['3.0','3.4'],
+            default_value => '3.4',
+        },
+        output_filter => {
+            is => 'Text',
+            doc =>
+                'Type of epitopes to report in the final output - select \'top\' to report the top epitopes in terms of fold changes,  \'all\' to report all predictions ',
+            valid_values => ['top', 'all'],
+        },
+        sample_name => {
+            is => 'Text',
+            doc => 'The sample name of the file being processed',
+            is_optional => 1,
+        },
+        variant_type => {
+            is => 'Text',
+            doc => 'The variant type being processed',
+            is_optional => 1,
+        }
+    ],
+};
+
+sub command_class_prefix {
+    return "Genome::Model::Tools::EpitopePrediction";
+}
+
+sub execute {
+    my $self = shift;
+
+    $self->debug_message("Validating Inputs...");
+    $self->_validate_inputs();
+
+    $self->debug_message("Constructing Workflow...");
+    my $workflow = $self->_construct_workflow();
+
+    $self->debug_message("Getting Workflow Inputs...");
+    my $inputs = $self->_get_workflow_inputs();
+
+    $self->debug_message("Running Workflow...");
+    my $result = $workflow->execute(inputs => $inputs);
+
+    unless($result){
+        $self->error_message( join("\n", map($_->name . ': ' . $_->error, @Workflow::Simple::ERROR)) );
```

Comment:
Oh, good to know. So this line can just go away? There is a similar line on the Pipeline.pm as well...